R/zchunk_L254.transportation_USA.R

```diff
@@ -87,101 +119,448 @@ module_gcam.usa_L254.transportation_USA_DISABLED <- function(command, ...) {
+
+    L254.tranSubsectorVOTT_USA %>%
       add_title("descriptive title of data") %>%
       add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+      add_comments("The same USA region values are repeated for each state") %>%
+      add_legacy_name("L254.tranSubsectorVOTT_USA") %>%
+      add_precursors("gcam-usa/states_subregions",
+                     "L254.tranSubsectorVOTT") ->
+      L254.tranSubsectorVOTT_USA
+
+    L254.tranSubsectorFuelPref_USA %>%
+      add_title("Subsector (fuel) preferences elasticity that are tied to GDP in the US states") %>%
+      add_units("Unitless") %>%
+      add_comments("The same USA region values are repeated for each state") %>%
+      add_comments("Fuel preferences are unrelated to time value") %>%
+      add_legacy_name("L254.tranSubsectorFuelPref_USA") %>%
+      add_precursors("gcam-usa/states_subregions",
+                     "L254.tranSubsectorFuelPref") ->
+      L254.tranSubsectorFuelPref_USA
+
+    L254.StubTranTech_USA %>%
+      add_title("Transportation stub technologies in the US states") %>%
+      add_units("NA") %>%
+      add_comments("The same USA region values are repeated for each state") %>%
+      add_legacy_name("L254.StubTranTech_USA") %>%
+      add_precursors("gcam-usa/states_subregions",
+                     "L254.StubTranTech") ->
+      L254.StubTranTech_USA
+
+    L254.StubTranTech_passthru_USA %>%
+      add_title("Transportation stub technologies for passthrough sectors in the US states") %>%
+      add_units("NA") %>%
+      add_comments("The same USA region values are repeated for each state") %>%
+      add_legacy_name(" L254.StubTranTech_passthru_USA") %>%
+      add_precursors("gcam-usa/states_subregions",
+                     "L254.StubTech_passthru") ->
+      L254.StubTranTech_passthru_USA
+
+    L254.StubTranTech_nonmotor_USA %>%
+      add_title("Transportation stub technologies for non-motorized subsectors in the US states") %>%
+      add_units("NA") %>%
+      add_comments("The same USA region values are repeated for each state") %>%
+      add_legacy_name("L254.StubTranTech_nonmotor_USA") %>%
+      add_precursors("gcam-usa/states_subregions",
+                     "L254.StubTech_nonmotor") ->
+      L254.StubTranTech_nonmotor_USA
+
+    L254.StubTranTechLoadFactor_USA %>%
+      add_title("Load factors of transportation stub technologies in the US states (all periods)") %>%
+      add_units("person/vehicle and tonnes/vehicle") %>%
+      add_comments("The same USA region values are repeated for each state") %>%
+      add_legacy_name("L254.StubTranTechLoadFactor_USA") %>%
+      add_precursors("gcam-usa/states_subregions",
+                     "L254.StubTranTechLoadFactor") ->
+      L254.StubTranTechLoadFactor_USA
+
+    L254.StubTranTechCost_USA %>%
+      add_title("Costs of transportation stub technologies in the US states (all periods)") %>%
```

Comment:
Why are we saying "all periods" here and in some other output table titles? Is it really required?